alacritty.yml

```diff
@@ -284,6 +290,13 @@
   #
   #indexed_colors: []
 
+  # Opaque backgrounds with transparency enabled
+  #
+  # Whether or not `background_opacity` applies to all backgrounds or only the
+  # default background. When set to `false` all cells will be transparent
+  # regardless of their background color.
+  #opaque_background_colors: true
```

Comment:
It has been a while and I don't completely remember our previous discussions, but `opaque_colored_backgrounds` might be an alternative name for this that is a little more clear? I'm not sure though, I think both would probably be fine.

Reply:
I don't have strong feelings either way. Whatever you prefer.

Reply:
Let's just leave it as-is for now. I don't really think it's a significant improvement.